app.rb

```diff
@@ -4,17 +4,24 @@
 
 class Chitter < Sinatra::Base
   get '/' do
-    # @peeps = [
-    #   "Hello World!",
-    #   "I am feeling sunny today!",
-    #   "Minions will takeover the world."
-    # ]
-    # @peeps.join
     p ENV
     @peeps = Peep.all
     erb :index
   end
 
+  get '/new-peep' do
+    erb :add_new_post
+  end
+
+  post '/post-peep'do
```

Comment:
Very well laid out, does what it says on the tin, but also is not confusing by using the word PEEP three hundred times. Good work.